framework/TApplication.php

```diff
@@ -1104,6 +1108,20 @@ public function onError($param)
 		$this->getErrorHandler()->handleError($this, $param);
 	}
 
+	/**
+	 * Raises onInitComplete event.
+	 * At the time when this method is invoked, application modules are loaded,
+	 * user request is resolved and the corresponding service is loaded and
+	 * initialized. The application is about to start processing the user
+	 * request.  This call is important for Shell applications that do not have
+	 * a web service stack.  This is the first and last call for finalization
+	 * of any loaded modules in Shell mode.
```

Comment:
Sorry for being this nitpick, but I find this a bit confusing for new users,  can you please reword this to make it more propositive, something like "In this event modules can perform any needed late initialization before the service starts to process the request" or something similar?